### PR TITLE
feat: Addition of new public theme tokens.

### PR DESCRIPTION
**Pull request by DaemonCahill**
### Description

This PR adds several new tokens to the public theme API in order to adjust the presentation of the `AppLayout`, `ContentLayout`, and `TopNavigation` components.

### How has this been tested?

<details>
   <summary>Review checklist</summary>

_The following items are to be evaluated by the author(s) and the reviewer(s)._

#### Correctness

- _Changes include appropriate documentation updates._
- _Changes are backward-compatible if not indicated, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#public-apis)._
- _Changes do not include unsupported browser features, see [`CONTRIBUTING.md`](https://github.com/cloudscape-design/components/blob/main/CONTRIBUTING.md#browsers-support)._
- _Changes were manually tested for accessibility, see [accessibility guidelines](https://cloudscape.design/foundation/core-principles/accessibility/)._

#### Security

- _If the code handles URLs: all URLs are validated through [the `checkSafeUrl` function](https://github.com/cloudscape-design/components/blob/main/src/internal/utils/check-safe-url.ts)._

#### Testing

- _Changes are covered with new/existing unit tests?_
- _Changes are covered with new/existing integration tests?_
</details>

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: style-dictionary/visual-refresh/metadata/colors.ts [hunk 1]
```diff
@@ -112,6 +112,36 @@ const metadata: StyleDictionary.MetadataIndex = {
     description:
       'The background color of app layout panel content area. For example: The side navigation and tools panel content background color.',
   },
+  colorBackgroundLayoutToggleActive: {
+    description: 'The background color ot the app layout toggle button when it is active.',
```
Comment (stirache) [inline style-dictionary/visual-refresh/metadata/colors.ts:116]: Suggested rephrase: 'The background color of the app layout toggle button when it's active.'

File: style-dictionary/visual-refresh/metadata/colors.ts [hunk 2]
```diff
@@ -112,6 +112,36 @@ const metadata: StyleDictionary.MetadataIndex = {
...
+    public: true,
+    themeable: true,
+  },
+  colorBackgroundLayoutToggleDefault: {
+    description: 'The default background color ot the app layout toggle button.',
```
Comment (stirache) [inline style-dictionary/visual-refresh/metadata/colors.ts:121]: Typo: 'The default background color _of_ the app layout toggle button.'

File: style-dictionary/visual-refresh/metadata/colors.ts [hunk 3]
```diff
@@ -112,6 +112,36 @@ const metadata: StyleDictionary.MetadataIndex = {
...
+    public: true,
+    themeable: true,
+  },
+  colorBackgroundLayoutToggleHover: {
+    description: 'The background color of the app layout toggle button when it is hovered.',
```
Comment (stirache) [inline style-dictionary/visual-refresh/metadata/colors.ts:126]: Suggested rephrase: 'The background color of the app layout toggle button on hover.'

File: style-dictionary/visual-refresh/metadata/colors.ts [hunk 4]
```diff
@@ -112,6 +112,36 @@ const metadata: StyleDictionary.MetadataIndex = {
...
+    public: true,
+    themeable: true,
+  },
+  colorBackgroundLayoutToggleSelectedActive: {
+    description: 'The background color of the app layout toggle button when it is selected and active.',
```
Comment (stirache) [inline style-dictionary/visual-refresh/metadata/colors.ts:131]: Replace "it is" with "it's"

File: style-dictionary/visual-refresh/metadata/colors.ts [hunk 5]
```diff
@@ -112,6 +112,36 @@ const metadata: StyleDictionary.MetadataIndex = {
...
+    public: true,
+    themeable: true,
+  },
+  colorBackgroundLayoutToggleSelectedDefault: {
+    description: 'The default background color of the app layout toggle button when it is selected.',
```
Comment (stirache) [inline style-dictionary/visual-refresh/metadata/colors.ts:136]: Replace "it is" with "it's"

File: style-dictionary/visual-refresh/metadata/colors.ts [hunk 6]
```diff
@@ -112,6 +112,36 @@ const metadata: StyleDictionary.MetadataIndex = {
...
+    public: true,
+    themeable: true,
+  },
+  colorBackgroundLayoutToggleSelectedHover: {
+    description: 'The background color of the app layout toggle button when it is selected and hovered.',
```
Comment (stirache) [inline style-dictionary/visual-refresh/metadata/colors.ts:141]: Suggested rephrase: 'The background color of the app layout toggle button when it's selected and on hover.'

File: style-dictionary/visual-refresh/metadata/colors.ts [hunk 7]
```diff
@@ -407,6 +437,11 @@ const metadata: StyleDictionary.MetadataIndex = {
     public: true,
     themeable: true,
   },
+  colorTextInteractiveActive: {
+    description: 'The color of clickable elements in their active state. For example: tabs, and icons.',
```
Comment (stirache) [inline style-dictionary/visual-refresh/metadata/colors.ts:441]: Remove comma after "tabs"

**Review by stirache — CHANGES_REQUESTED** [review]

**Review by stirache — DISMISSED** [review]

File: style-dictionary/visual-refresh/metadata/colors.ts [hunk 8]
```diff
@@ -112,6 +112,36 @@ const metadata: StyleDictionary.MetadataIndex = {
...
+    public: true,
+    themeable: true,
+  },
+  colorBackgroundLayoutToggleSelectedDefault: {
+    description: "The default background color of the app layout toggle button when it's selected.",
+    public: true,
+    themeable: true,
+  },
+  colorBackgroundLayoutToggleSelectedHover: {
+    description: "The background color of the app layout toggle button when it's selected and on hover.",
```
Comment (gethinwebster) [inline style-dictionary/visual-refresh/metadata/colors.ts:141]: "selected and on hover" doesn't read right to me, what do you think @stirache ?

**Review by gethinwebster — CHANGES_REQUESTED** [review]

**Review by gethinwebster — APPROVED** [review]

**Pull request merged**